scrunch/helpers.py

```diff
@@ -7,6 +7,12 @@
     from urllib.parse import urljoin
 
 
+DEFAULT_CATEGORIES = [
```

Comment:
Maybe rename this to `DEFAULT_MULTIPLE_RESONSE_CATEGORIES` or something to make it a bit more clear that this is for MR vars